modules/core/src/lib/attribute-manager.js

```diff
@@ -232,7 +232,9 @@ export default class AttributeManager {
     for (const attributeName in this.attributes) {
       const attribute = this.attributes[attributeName];
 
-      if (
+      if (!attribute.isEnabled(context)) {
```

Comment:
Nit: Not sure that hiding the callback in this method makes this more readable...